backend/src/web-server/controllers/north-connector.controller.ts

```diff
@@ -539,4 +546,274 @@ export default class NorthConnectorController {
+    this.getManifestWithItemsMode(ctx);
+    const northItems = ctx.app.repositoryService.northItemRepository.listNorthItems(ctx.params.northId);
+    ctx.ok(northItems);
+  }
+
+  async searchNorthItems(ctx: KoaContext<void, Page<NorthConnectorItemDTO>>): Promise<void> {
+    this.getManifestWithItemsMode(ctx);
+    const searchParams: NorthConnectorItemSearchParam = {
+      page: ctx.query.page ? parseInt(ctx.query.page as string, 10) : 0,
+      name: (ctx.query.name as string) || null
+    };
+    const northItems = ctx.app.repositoryService.northItemRepository.searchNorthItems(ctx.params.northId, searchParams);
+    ctx.ok(northItems);
+  }
+
+  async northItemsToCsv(ctx: KoaContext<{ items: Array<NorthConnectorItemDTO> }, any>): Promise<void> {
+    this.getManifestWithItemsMode(ctx);
+
+    const northItems = ctx.request.body!.items.map(item => {
+      const flattenedItem: Record<string, any> = {
+        ...item
+      };
+      for (const [itemSettingsKey, itemSettingsValue] of Object.entries(item.settings)) {
+        if (typeof itemSettingsValue === 'object') {
+          flattenedItem[`settings_${itemSettingsKey}`] = JSON.stringify(itemSettingsValue);
+        } else {
+          flattenedItem[`settings_${itemSettingsKey}`] = itemSettingsValue;
+        }
+      }
+      delete flattenedItem.id;
+      delete flattenedItem.settings;
+      delete flattenedItem.connectorId;
+      return flattenedItem;
+    });
+    ctx.body = csv.unparse(northItems);
+    ctx.set('Content-disposition', 'attachment; filename=items.csv');
+    ctx.set('Content-Type', 'application/force-download');
+    ctx.ok();
+  }
+
+  async exportNorthItems(ctx: KoaContext<any, any>): Promise<void> {
+    this.getManifestWithItemsMode(ctx);
+
+    const northItems = ctx.app.repositoryService.northItemRepository.getNorthItems(ctx.params.northId).map(item => {
+      const flattenedItem: Record<string, any> = {
+        ...item
+      };
+      for (const [itemSettingsKey, itemSettingsValue] of Object.entries(item.settings)) {
+        if (typeof itemSettingsValue === 'object') {
+          flattenedItem[`settings_${itemSettingsKey}`] = JSON.stringify(itemSettingsValue);
+        } else {
+          flattenedItem[`settings_${itemSettingsKey}`] = itemSettingsValue;
+        }
+      }
+      delete flattenedItem.id;
+      delete flattenedItem.settings;
+      delete flattenedItem.connectorId;
+      return flattenedItem;
+    });
+    ctx.body = csv.unparse(northItems);
+    ctx.set('Content-disposition', 'attachment; filename=items.csv');
+    ctx.set('Content-Type', 'application/force-download');
+    ctx.ok();
+  }
+
+  async checkImportNorthItems(ctx: KoaContext<{ itemIdsToDelete: string }, any>): Promise<void> {
+    const manifest = this.getManifestWithItemsMode(ctx);
+
+    const file = ctx.request.file;
+    if (file.mimetype !== 'text/csv') {
+      return ctx.badRequest();
+    }
```

Comment:
I actually had to remove this because on some Windows system, it was not working properly. Let's discuss that